packages/nimble-components/src/utilities/style/display.ts

```diff
@@ -10,4 +10,4 @@ import {
  */
```

Comment:
Could you update the docs here and in the spright version of this file to mention the new behavior?

There's also a mention of it in `packages/nimble-components/docs/css-guidelines.md`